tests/test_shell.py

```diff
@@ -269,9 +273,6 @@ def test_clone(self):
         job.init()
         assert len(project_a) == 1
         assert len(project_b) == 0
-        with pytest.raises(ExitCodeError):
-            self.call("python -m signac clone {} 9bfd29df07674bc5"
-                      .format(os.path.join(self.tmpdir.name, 'b')).split())
```

Comment:
Why was this removed? This seems like the negative test corresponding to the positive test below.

Reply:
This test was not removed, I just changed its position. I was trying to arrange the test in a certain order (like first do all positive tests then negative tests).